components/sections/home-page/GitHubMock.tsx

```diff
@@ -87,7 +88,5 @@ const GitHubMock: FC<GitHubMockProps> = ({ moreHeading, topFeature }) => {
         </div>
       </div>
     </SectionWrapper>
-  )
```

Comment:
@CBID2 you need to add this back in.

Reply:
I'm not near my computer yet @Arol15. Can you add the parentheses as a suggestion?

Reply:
Unfortunately, GitHub doesn't allow adding suggestions on deleted code.

Reply:
Ok I'll do it once I get on there